pkg/client/oauthaccesstoken.go

```diff
@@ -25,6 +26,13 @@ func newOAuthAccessTokens(c *Client) *oauthAccessTokenInterface {
 	}
 }
 
+// Get returns information about a particular image and error if one occurs.
+func (c *oauthAccessTokenInterface) Get(name string) (result *oauthapi.OAuthAccessToken, err error) {
```

Comment:
I use it to retrieve token contents when outputting token info: https://github.com/juanvallejo/origin/blob/jvallejo_add-cli-oauth-describer/pkg/cmd/cli/describe/describer.go#L468